common/tools/dev-tool/README.md

```diff
@@ -20,6 +20,7 @@ It provides a place to centralize scripts, resources, and processes for developm
     - `dev` (link samples to local sources for access to IntelliSense during development)
     - `prep` (prepare samples for local source-linked execution)
     - `run` (execute a sample or all samples within a directory)
+    - `checkNodesVers` (execute samples with different node versions, typically in preparation for release)
```

Comment:
I'd love to be more verbose here for clarity - `checkNodeVersions` or something like that

Reply:
All commands here are abbreviated (e.g. `dev`, `prep`, etc) so I followed that convention.

Reply:
That's fair, but I still think that `dev` and `prep` are kind of unambiguous (maybe less so dev) but `vers` is less clear - version, versions, verbosity (for some reason). It's not a well-known abbrev.

Reply:
`dev` is probably the worst command name I've ever come up with, and that was only because I didn't want to call it `create-recursive-development-symlink`, so I would be careful about using it as a standard :stuck_out_tongue:

Reply:
ok, I renamed it to `check-node-versions`.